src/test/java/ru/training/at/hw3/page/DifferentElementsPage.java

```diff
@@ -0,0 +1,69 @@
+package ru.training.at.hw3.page;
+
+import ru.training.at.hw3.utils.WaitActions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import javax.inject.Inject;
+import java.util.List;
+
+public class DifferentElementsPage {
+
+    private WaitActions waitActions;
+    private WebDriver webDriver;
+    private WebElement webElement;
+
+
+    @FindBy(css = ".label-checkbox:first-child > input")
+    private WebElement waterCheckbox;
+
+    @FindBy(css = ".label-checkbox:nth-last-child(2) > input")
+    private WebElement windCheckbox;
+
+    @FindBy(css = ".label-radio:last-child > input")
+    private WebElement selenRadio;
+
+    @FindBy(css = ".colors")
+    private WebElement colorsDropDown;
+
+    @FindBy(xpath = "//*[text()='Yellow']")
+    private WebElement yellow;
+
+    @FindBy(css = "ul.panel-body-list > li")
+    private List<WebElement> logs;
+
+
+    @Inject
```

Comment:
What if the purpose of the using of current annotation?

Reply:
If I don't use this annotation I have a com.google.inject.ConfigurationException: Guice configuration errors and as a result only the FirstTest runs well (the Home page one), and the 2nd (related to this page - DifferentElementsPage) fails.
I am not sure I understand this magic completely, but I believe this is how I have the driver here on this page.